.changeset/few-carpets-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
```

Comment:
Even though this would normally be a minor change, given that this is pre 1.0 doing a minor bump would require users to update their package.json (if they are using `^`), so we are doing patch even for new features.

Reply:
Ah, right! I forgot this is pre 1.0. I'll get this updated!